### PR TITLE
French translation for the `path` CSS function reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR creates a French translation for the `path` CSS function reference.

### CODE REVIEW

File: files/fr/web/css/path()/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">path( [[&lt;'fill-rule'&gt;,]?&lt;string&gt;)</pre>
+
+<h3 id="parameters">Paramètres</h3>
+
+<dl>
+ <dt><code>&lt;'fill-rule'&gt;</code></dt>
+ <dd>La règle de remplissage de l'intérieur du tracé. Les valeurs possibles sont <i>nonzero</i> ou <i>evenodd</i>. La valeur par défaut est <i>nonzero</i>. Voir <a href="/fr/docs/Web/SVG/Attribute/fill-rule">fill-rule</a> pour plus de détails.</dd>
+ <dt>&lt;string&gt;</dt>
```
Comment (SphinxKnight) [inline files/fr/web/css/path()/index.html:19]: to be enclosed between \<code> (even if en-US doesn't (yet))

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
One minor stuff before merging :)
Thanks again :)

File: files/fr/web/css/path()/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/path()/index.html:19]: Honestly I hesitated ^^
Fixed in https://github.com/mdn/translated-content/pull/1747/commits/fb717b0df16bb78f0faa9ccc5d5377ceb56533bc

Comment (SphinxKnight) [reply files/fr/web/css/path()/index.html:19]: Oops :) I was actually seeing the &amp;lt;string&amp;gt; in the second &lt;dt&gt; and did not spot the earlier code words.
Could you add another pair of &lt;code&gt;? (You can leave fb717b0 as is, it's fine :) )

**Review by SphinxKnight — CHANGES_REQUESTED** [review]

**Comment by audrasjb** [comment]
Ah yes, sure!
Fixed in https://github.com/mdn/translated-content/pull/1747/commits/533c9d994a89fa13dbd6dfef60aad8e9db553270

**Review by SphinxKnight — APPROVED** [review]

**Pull request merged**